docs/src/starknet/101.md

```diff
@@ -72,6 +72,7 @@ Read more about accounts [here](./account-import.md)
 
 Calling a contract doesn't require an account
 
+<!-- { "ignored_output": true } -->
```

Comment:
Why don't we validate output?

Reply:
We use the same contract in the example below with invoke, which modifies the contract storage. If anyone ever calls that contract (which seems to have happened), the value returned here will be different than we assert.